### ISSUE TITLE
error during command line import on large channels

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#6597]
### Observed behavior

Reported from a user:

```
kolibri manage importchannel -- network a9b25ac9814742c883ce1b0579448337
kolibri manage importcontent -- network a9b25ac9814742c883ce1b0579448337
```

This method works great on small channels (< say ~2GB give or take).  Anything too much larger get a Python error:  "OverflowError: Python int too large to convert to C long".

This works via the GUI

### Expected behavior

Command-line import works for all channel sizes

### User-facing consequences

complications with automated provisioning

### Steps to reproduce

…

### Context

* 0.13
* Ubuntu 16.04

**Comment by jamalex** [comment]
It looks likely this is related to 32bit Python; if this is confirmed, then we should also make sure to add 32bit to the automated test suite envs, if possible.

**Comment by indirectlylit** [comment]
Traceback here:

```
Traceback (most recent call last):
  File "/usr/local/bin/kolibri", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/kolibri/dist/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/kolibri/dist/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/kolibri/dist/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/kolibri/utils/cli.py", line 243, in invoke
    return super(KolibriDjangoCommand, self).invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/kolibri/dist/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/kolibri/dist/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/kolibri/dist/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/kolibri/utils/cli.py", line 641, in manage
    execute_from_command_line(["kolibri manage"] + ctx.args)
  File "/usr/local/lib/python2.7/dist-packages/kolibri/dist/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python2.7/dist-packages/kolibri/dist/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python2.7/dist-packages/kolibri/dist/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python2.7/dist-packages/kolibri/dist/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python2.7/dist-packages/kolibri/core/tasks/management/commands/base.py", line 96, in handle
    return self.handle_async(*args, **options)
  File "/usr/local/lib/python2.7/dist-packages/kolibri/core/content/management/commands/importcontent.py", line 434, in handle_async
    renderable_only=options["renderable_only"],
  File "/usr/local/lib/python2.7/dist-packages/kolibri/core/content/management/commands/importcontent.py", line 159, in download_content
    renderable_only=renderable_only,
  File "/usr/local/lib/python2.7/dist-packages/kolibri/core/content/management/commands/importcontent.py", line 253, in _transfer
    total=total_bytes_to_transfer
  File "/usr/local/lib/python2.7/dist-packages/kolibri/core/tasks/management/commands/base.py", line 109, in start_progress
    total=total, level=level, update_callback=self._update_all_progress
  File "/usr/local/lib/python2.7/dist-packages/kolibri/core/tasks/management/commands/base.py", line 32, in __init__
    self.progressbar = click.progressbar(length=total, width=0)
  File "/usr/local/lib/python2.7/dist-packages/kolibri/dist/click/termui.py", line 351, in progressbar
    width=width, color=color)
  File "/usr/local/lib/python2.7/dist-packages/kolibri/dist/click/_termui_impl.py", line 81, in __init__
    iterable = range_type(length)
OverflowError: Python int too large to convert to C long
```

Click progress bars were reinstated in https://github.com/learningequality/kolibri/pull/5927

**Comment by indirectlylit** [comment]
I believe this is an issue with `xrange` in Python 2. The easiest solution is likely to disable progress bars in Py2

**Comment by indirectlylit** [comment]
hopefully fixed by  #6630
